pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -160,3 +205,47 @@ def _luvoir_matrix_single_mode(number_all_modes, wfe_aber, luvoir_sim, resDir, s
         plt.savefig(os.path.join(resDir, 'OTE_images', opd_name + '.pdf'))
 
     return efield_focal_plane
+
+
+def _rst_matrix_single_mode(wfe_aber, rst_sim, resDir, saveefields, saveopds, mode_no):
+    '''
+    Function to calculate RST Electrical field (E_field) of one DM actuator in CGI.
+    :param wfe_aber: float, amplitude injected inside the chosen actuator
```

Comment:
**Always** specify input units --> "float, calibration aberration per actuator in m" (correct the m if that is not true)